### DISCUSSION TITLE
Lesson 5: IndexError: list index out of range (in read_value.py)

### DISCUSSION THREAD

**Discussion by LinusWeigand** [smartcontractkit/full-blockchain-solidity-course-py#1235] [Q&A] [answered]
I keep getting this error when executing read_value.py. Anyone know what to do about it? The error message reads:

Running 'scripts/read_value.py::main'...
  File "brownie/_cli/run.py", line 51, in main
    return_value, frame = run(
  File "brownie/project/scripts.py", line 103, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File "./scripts/read_value.py", line 10, in main
    read_contract()
  File "./scripts/read_value.py", line 5, in read_contract
    simple_storage = SimpleStorage[-1]
  File "brownie/network/contract.py", line 175, in __getitem__
    return self._contracts[i]
IndexError: list index out of range
Terminating local RPC client...

**Comment by LinusWeigand** [comment, marked as answer]
Solved it myself how can I delete a post?

**Reply by cromewar** [reply]
Actually could you solve the solution? this could help the community a lot ;)
